src/LinearSolvers/iterativeWrapper.jl

```diff
@@ -24,6 +24,7 @@ Example:
 
 """
 type IterativeSolver<: AbstractSolver
+   AA   # matrix being solved
```

Comment:
Why do we need this field exactly? As we've discussed I think its better to have AA being stored in the `pFor`. One reason is that the iterative solver doesn't in general know how to generate the PDE if `AA` is empty. Please take a look at my changes to `DivSigGrad`. You can do similar things for maxwell codes.